### PR TITLE
Filter-out http-streams.

### PR DESCRIPTION
**Pull request by relictMarauder**
Filter-out the Http-Streams by selecting quality.

### CODE REVIEW

File: src/resources/lib/modeling.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def alphanum_key(key: str) -> List[Union[str, int]]:
         flatten_urls_dict = {
             f"{quality}@{stream}": url
             for quality, urls in files.items()
-            for stream, url in urls.items()
+            for stream, url in list(filter(lambda x: (x[0] != "http"), urls.items()))
```
Comment (quarckster) [inline src/resources/lib/modeling.py:272]: This dictionary comprehension gets too complicated. Let's split it into multiple loops for better readability.

**Comment by quarckster** [comment]
By the way, I thought that Kinopub deprecated http streams and I removed this type of streaming in https://github.com/quarckster/kodi.kino.pub/pull/260

**Comment by relictMarauder** [comment]
> By the way, I thought that Kinopub deprecated http streams and I removed this type of streaming in #260

Yes, but the http stream is still in the list of urls, which is not in sync with another parts of add-on.

**Comment by quarckster** [comment]
Superseded by #292.

**Pull request closed without merging**